### DISCUSSION TITLE
Is Dialogic 1 compatible with Godot 4?

### DISCUSSION THREAD

**Discussion by itajuniorart** [dialogic-godot/dialogic#1738] [Q&A] [answered]
Hi guys, could someone help me out? I'm currently working with Godot 4, but I'm hesitant to use Dialogic 2 since it's still in alpha. I was wondering if Dialogic 1 is compatible with Godot 4. 

If not, is Dialogic 2 stable enough? 
It's just for a small solo indie game project, with 150 maybe 250 dialogues.

**Comment by zaknafean** [comment, marked as answer]
Unfortunately Dialogic 1.x is only for Godot 3.x branch. There are no plans to port it to godot 4 or to back port Dialogic 2 to Godot 3.

Godot 3.x = Dialogic 1
Godot 4.x = Dialogic 2


That said dialogic 2 is very close to beta. If you're gonna use it I suggest you join the discord for support.

**Reply by itajuniorart** [reply]
I understand. So I'm going to try Dialogic 2 and have already joined Discord. Thank you for clearing my doubts.
